packages/shared/package-lock.json

```diff
@@ -0,0 +1,6373 @@
+{
```

Comment:
Please remove this package-lock and run `npm run bootstrap:dev` in the root :pray: